components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -521,7 +524,10 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 	}
 
 	__onViewResize(e) {
-		this.style.height = `${e.detail.height}px`;
```

Comment:
The event delay above is the actual fix for the `ResizeObserver` error, but when investigating I also found this was sometimes being set to the same value multiple times